builtin/game/misc.lua

```diff
@@ -4,74 +4,42 @@
 -- Misc. API functions
 --
 
-local timers = {}
-local mintime
-local function update_timers(delay)
-	mintime = false
-	local sub = 0
-	for index = 1, #timers do
-		index = index - sub
-		local timer = timers[index]
-		timer.time = timer.time - delay
-		if timer.time <= 0 then
-			core.set_last_run_mod(timer.mod_origin)
-			timer.func(unpack(timer.args or {}))
-			table.remove(timers, index)
-			sub = sub + 1
-		elseif mintime then
-			mintime = math.min(mintime, timer.time)
-		else
-			mintime = timer.time
-		end
-	end
-end
+local _jobs = {}
+local _time = 0.0
+local _last = 0.0
```

Comment:
habit, it makes spotting locals a bit more apparent. In various C styles, _ and __ are using to delimit "internal" stuff that's not supposed to leak or is available out of context. I can remove these, obviously.